packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,212 @@
 # @fluentui/react-persona Spec
 
+Convergence epic issue: #24213
+
 ## Background
 
-_Description and use cases of this component_
+A Persona is a visual representation of a person or status that showcases an Avatar, PresenceBadge, or an Avatar with a PresenceBadge. This composite component is not a card, therefore it will not provide a visual board or a popup when the user hovers the Avatar.
+
+Persona is used in PeoplePicker, Team's left rail and menus, chiclets, and Card.
 
 ## Prior Art
 
-_Include background research done for this component_
+- OpenUI research: [Avatar/Persona](https://open-ui.org/components/avatar.research)
+- v7/v8: [link](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)
+  - > Note: v8's Persona is a combination of v9's Persona and Avatar (v8's Persona has a sub-component PersonaCoin that would count as Avatar, but is not exposed.). In v9 we've taken the approach of having a single component Avatar and another component Persona.
+- v0: Does not have equivalent component.
+
+## Anatomy
+
+![Anatomy](./etc/images/anatomy.png)
+
+### Avatar, PresenceBadge, and Avatar + PresenceBadge
+
+v8's Persona has a sub-component `PersonaCoin`, the equivalent in v9 is an Avatar. Currently Avatar is able to display a PresenceBadge, but this behavior will be deprecated in favor of Persona having this behavior.
+
+Persona has a slot for an `Avatar` and `PresenceBadge`, giving the user the ability to display an Avatar, a PresenceBadge, or Combine them into an Avatar with presence.
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### Text lines
+
+Other than styling and naming, the text lines in Persona remain the same.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+![Avatar](./etc/images/avatar.png)
+
+Persona with Avatar:
+
+```jsx
+<Persona avatar={{ name: 'Kevin Sturgis' }} primaryText="Kevin Sturgis" secondaryText="Software Engineer" />
+```
+
+![PresenceBadge](./etc/images/badge.png)
+
+Persona with PresenceBadge:
+
+```jsx
+<Persona presence={{ status: 'offline', outOfOffice: true }} primaryText="Kevin Sturgis" />
+```
+
+![Avatar and PresenceBadge](./etc/images/avatar_badge.png)
+
+Persona with Avatar + PresenceBadge:
+
+```jsx
+<Persona
+  avatar={{ name: 'Kevin Sturgis' }}
+  presence={{ status: 'offline', outOfOffice: true }}
+  primaryText="Kevin Sturgis"
+  secondaryText="Software Engineer"
+  tertiaryText="Offline"
+/>
```

Comment:
What do you think about having the `avatar` slot be the primary slot? That would have a few benefits:

- Somewhat cleaner API since the Avatar props like `name` and `image` won't be nested inside a separate prop.
- This is how the Field controls work, and Fields conceptually do a similar thing: adding extra text and layout to an existing control.
- The value of `primaryText` could default to the `name` prop, to reduce the need to duplicate the name in multiple places.
- The `size` prop can be used to determine the text size (for the "fixed" variant). Or, the number of lines of text can be used to determine the `size` prop (for the "scaled" variant). You could even determine whether it's "fixed" or "scaled" based on whether or not the `size` prop was supplied, without relying on a separate prop.

That creates a few open questions:
1. What to do with the `presence` prop in this spec, since Avatar has a `badge` prop for the same purpose. One option would be to remove `presence` and just use `badge`. Another option is to Omit the `badge` prop from the primary slot, and instead add a `presence` slot.
2. How to pick the "presence badge" variant. One option is another prop like `badgeOnly` or `display="badge"` or something.

Reply:
I agree, making avatar the primary slot will make things simpler.

For the open questions:
1. If we go for simplicity, letting Avatar deal with the badge would be best. Imo with the change of Avatar becoming the primary slot, it feels like presence should have its own slot. This way it would also make it easier to deal with the presence alone scenario, otherwise we would need to extract the badge from Avatar.
2. I would say badgeOnly would be the best option since I feel like display means that there will be more than two display types.

TLDR, what I think we should do is:
1. Make Avatar the primary slot and omitting its badge prop.
2. Keep the slot for presence.
3. Add `badgeOnly` prop to deal with presence variant.

What do you think?

Reply:
After I thought a little more about it, this could bring some problems with the slot types. If we make Avatar the primary slot, it would be NonNullable and therefore the presence only variant will have a weird API.

Reply:
@behowell quick update, I added these changes to this PR #24749. I think it works well, but let me know what you think about it!